web-server/web-server.markdown

```diff
@@ -110,16 +110,17 @@ and other codes have other meanings.
 The status phrase repeats that information in a human-readable phrase like "OK" or "not found".
 
 For the purposes of this chapter
-there are only two other things we need to know about HTTP
+there are only two other things we need to know about HTTP.
+
 The first is that it is *stateless*:
 each request is handled on its own,
 and the server doesn't remember anything between one request and the next.
 If an application wants to keep track of something like a user's identity,
 it must do so itself.
+
 The usual way to do this is with a cookie,
-which is just a short character string that the server sends to the client,
+which is a short character string that the server sends to the client,
 and the client later returns to the server.
-
 When a user signs in,
```

Comment:
"signs in" here implies that the user is actually logging in to something, but servers don't need you to log in in order to create cookies -- suggest changing to "When a user visits a page," or something similar.